### PR TITLE
DP-9130 add inputTextTypeahead Org dropdown to SearchBanner

### PR DESCRIPTION
**Pull request by pmallett-mc**
Jira ticket --> https://jira.mass.gov/browse/DP-9130

### CODE REVIEW

**Comment by pmallett-mc** [comment]
This is ready for internal review; now with mobile styles:
with-mobile-styles

File: src/components/organisms/SearchBanner/index.js [hunk 1]
```diff
@@ -9,24 +12,48 @@ class SearchBanner extends Component {
...
       const searchInput = nextProps.searchBox.defaultText;
       this.setState({ searchInput });
     }
   }
+  toggleFilterBox() {
+    this.setState((prevState) => ({ filterBoxExpanded: !prevState.filterBoxExpanded }));
+  }
+  filterBoxSubmitOnClick() {
+    this.setState((prevState) => ({ filterBoxExpanded: !prevState.filterBoxExpanded }));
+    this.outerSubmitButtonOnClick();
```
Comment (smurrayatwork) [inline src/components/organisms/SearchBanner/index.js]: This only exists if props.filterBox does. You should check for if this function exists.

Comment (avrilpearl) [reply src/components/organisms/SearchBanner/index.js]: if( typeof this.outerSubmitButtonOnClick() === 'function' ) {
 this.outerSubmitButtonOnClick()
)

Comment (pmallett-mc) [reply src/components/organisms/SearchBanner/index.js]: Updated in 911f6d2

**Comment by clairesunstudio** [comment]
need to also follow up on:
screen shot 2018-06-05 at 1 51 47 pm

**Pull request merged**